src/patternfly/components/Card/card.scss

```diff
@@ -108,57 +136,128 @@
   --pf-c-card__header--m-toggle-right--toggle--MarginRight: calc(var(--pf-global--spacer--form-element) * -1);
   --pf-c-card__header--m-toggle-right--toggle--MarginLeft: var(--pf-global--spacer--xs);
 
-  // Card input
+  // Card input (selectable raised)
   --pf-c-card__input--focus--BorderWidth: var(--pf-global--BorderWidth--md);
   --pf-c-card__input--focus--BorderColor: var(--pf-global--primary-color--100);
 
+  position: relative;
   display: flex;
   flex-direction: column;
   background-color: var(--pf-c-card--BackgroundColor);
   box-shadow: var(--pf-c-card--BoxShadow);
 
-  &.pf-m-hoverable {
-    &:hover {
-      box-shadow: var(--pf-c-card--m-hoverable--hover--BoxShadow);
-    }
+  //////
+  // SELECTABLE CARDS, CLICKABLE CARDS
+  &.pf-m-selectable,
+  &.pf-m-clickable {
+    isolation: isolate;
+
+    // box-shadow will come from the card__label::before instead of the card so remove this so there's isn't a double shadow
+    box-shadow: none;
   }
 
-  &.pf-m-selectable {
-    position: relative;
+  .pf-c-card__label {
+    position: absolute;
+    inset: 0;
     cursor: pointer;
 
-    &:hover {
-      box-shadow: var(--pf-c-card--m-selectable--hover--BoxShadow);
+    &::before {
+      position: absolute;
+      inset: 0;
+      z-index: -1;
+      content: '';
+      background-color: var(--pf-c-card--BackgroundColor, transparent);
+      border: var(--pf-c-card--m-selectable--BorderWidth) solid var(--pf-c-card--BorderColor, transparent);
+      box-shadow: var(--pf-c-card--BoxShadow, none);
     }
+  }
 
-    &:focus {
-      box-shadow: var(--pf-c-card--m-selectable--focus--BoxShadow);
+  // Hover the card (label)
+  .pf-c-card__label:hover {
+    --pf-c-card--BackgroundColor: var(--pf-c-card--m-selectable--hover--BackgroundColor);
+    --pf-c-card--BoxShadow: var(--pf-c-card--m-selectable--hover--BoxShadow);
+  }
+
+  // Selected card
+  .pf-c-card__input:where(:checked) ~ .pf-c-card__label,
+  &.pf-m-selected {
```

Comment:
This implies needing to be nested in `.pf-c-card`, but you could either make this `.pf-c-card.pf-m-selected` or put an `@at-root` at-rule before the `.pf-c-card__input:where(...` line, which will compile to `.pf-c-card__input:where(...` instead of `.pf-c-card .pf-c-card__input:where(...`